test/studies/hpcc/common/bradc/unit/parRandom2-domblock.chpl

```diff
@@ -13,13 +12,13 @@ var B: [D] real;
 var randStr1 = new RandomStream(314159265);
 var randStr2 = new RandomStream(314159265);
 
-forall (i,r) in (A.domain, randStr1) do
+forall (i,r) in zip(A.domain, randStr1) do
   A(i) = r;
 
 for b in B do
   b = randStr2.getNext();
 
-for (i,a,b) in (D,A,B) {
+for (i,a,b) in zip(D,A,B) {
```

Comment:
I call it out in the commit message -- the 64-bit version did have format calls, and required similar changes, so they seemed more akin than not.